_data/enterprise-landing-page-tabs-eks-generatecode.yml

```diff
@@ -93,3 +87,68 @@
       </code>
       </pre>
       <figure>
+
+- id: app-deploy-code
+  title: sample-app/main.tf
```

Comment:
This is not what customers would get in the new world, as Argo and Flux do not support terraform. They would actually get something like this:

https://github.com/yorinasub17/gruntwork-sample-app-deploy/tree/main/aws-sample-app

Also, this conflicts with Step 5.

Reply:
Ohhh, good point, that completely overlaps! OK, will switch to VPC code snippet instead...

Reply:
Updated! https://github.com/gruntwork-io/gruntwork-io.github.io/pull/596/commits/d9c6cd3527bae1ea1af1ac16e7bdc7c7ac2037cb